src/controllers/PendingTransactionController.ts

```diff
@@ -0,0 +1,52 @@
+
+import { Request, Response } from "express";
+import { sendJSONresponse } from "../common/Utils";
+import { Config } from "../common/Config";
+import { Error } from "mongoose";
+let Promise = require("bluebird");
+
+export class PendingTransactionController {
+
+    public readPendingTransaction(req: Request, res: Response) {
+        const validationErrors: any = PendingTransactionController.validateQueryParameters(req);
+
+        if (validationErrors) {
+            sendJSONresponse(res, 400, {validationErrors});
+            return;
+        }
+
+        const address: string = req.query.address;
+
+        Config.web3.eth.getBlock("pending", true)
+        .then((pendings: any) => {
```

Comment:
Moreve `then` next to getBlock, much easier to read that way

Reply:
Done